kodilla-hibernate/src/main/java/com/kodilla/hibernate/manytomany/facade/FindFacadeException.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.hibernate.manytomany.facade;
+
+public class FindFacadeException extends Exception {
```

Comment:
I am not convinced to this class. "FindFacadeException" informs only that this exception was thrown from facade.
I rather expect exception EmployeeNotFoundException, or even more generic NotFoundException